lib/util/support.js

```diff
@@ -34,6 +34,13 @@ function isSupported() {
     return false;
   }
 
+  // NOTE (joma): Return if iOS chrome version is not 14.3 and above.
+  if (isChromeIOS()) {
+    if (!isChromeVersionSupported()) {
```

Comment:
```suggestion
    if (!isIOSVersionSupported()) {
```

Reply:
@manjeshbhargav Hm, I've removed this change in a recent commit because you mentioned that `isWebRTCSupported` will catch this. Do you think we should add an `iOSVersionSupported` check still?